css/style.css

```diff
@@ -1,289 +0,0 @@
-@import "/css/reset.css";
-@import "/css/variables.css";
-
-/* Mobile 375px Desktop 1440px */
-
-body {
-  box-sizing: content-box;
-  font-family: var(--Font-Family);
-  font-weight: var(--Font-Weight);
-  font-size: var(--Font-Size);
```

Comment:
**suggestion:** You can use shorthand here (https://www.w3schools.com/css/css_font_shorthand.asp)

Reply:
That'll help a lot!